### PR TITLE
Fix casper build

### PR DESCRIPTION
**Pull request by shaomeng**
This PR fixes a few compilation errors that occur on Casper.

### CODE REVIEW

**Comment by clyne** [comment]
@stasj, can you explain how "site.NCAR" is found? At one point we sym-linked site.local to site.NCAR. I haven't done that in a while, yet things seem to work fine...I'm a little confused.

**Comment by StasJ** [comment]
> @StasJ, can you explain how "site.NCAR" is found? At one point we sym-linked site.local to site.NCAR. I haven't done that in a while, yet things seem to work fine...I'm a little confused.

That was the case at one point but we moved the file into our repo.

File: site_files/site.NCAR [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
-    set (CMAKE_PREFIX_PATH    ${QTDIR}/lib/cmake )
-    set (Qt5Core_DIR          ${QTDIR}/lib/cmake/Qt5Core )
-    set (QT_QMAKE_EXECUTABLE  ${QTDIR}/gcc_64/bin/qmake )
-    set (OSPRAYDIR            ${THIRD_PARTY_DIR}/Ospray )
+    cmake_host_system_information( RESULT hostName QUERY HOSTNAME              )
+    if ( ${hostName} MATCHES "casper" )
+        message( "FOUND CASPER" )
+    	set (THIRD_PARTY_DIR /glade/p/cisl/vast/vapor/third-party/2020-Apr      )
+        set (QTDIR /lib )
+        #set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -stdlib=/glade/u/apps/dav/opt/gnu/9.1.0/lib64/libstdc++")
```
Comment (StasJ) [inline site_files/site.NCAR:12]: Do we need this?

Comment (shaomeng) [reply site_files/site.NCAR]: No, thanks for spotting it. There's not site file change now.

**Review by StasJ — APPROVED** [review]

File: site_files/site.NCAR (hunk 1 above)
Comment (StasJ) [reply site_files/site.NCAR]: I was referring to the commented out CXX flags but that’s fine

**Review by clyne — APPROVED** [review]

**Pull request merged**